### PR TITLE
Add Aums Work keyboard

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add aumz work keyboard, 65% multi layout with 2 version. hotswap and soldered, support via.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jaykeeb/aumz_work/hotswap/info.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                {"label": "GUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "Alt", "matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"label": "Ctrl", "matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "\u2190", "matrix": [4, 12], "x": 13, "y": 4},
+                {"label": "\u2193", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "\u2192", "matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
```
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/hotswap/info.json:106]: matching layouts condensed into one, `"layout_aliases"` used to conform to PR Checklist regarding more than one physical layout.
```suggestion
    },
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_65_ansi_blocker"
    },
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_tsangan"],
    "layouts": {
```

File: keyboards/jaykeeb/aumz_work/soldered/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,  KC_HOME,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
+        KC_LSFT, KC_BSLS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
```
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/soldered/keymaps/via/keymap.c:11]:
```suggestion
        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/jaykeeb/aumz_work/hotswap/info.json (hunk 1 above)
Comment (alabahuy) [reply keyboards/jaykeeb/aumz_work/hotswap/info.json]: oke done @dunk2k :)

**Comment by alabahuy** [comment]
oke i move similarities info json to keyboard level @waffle87 :)

**Review by drashna — APPROVED** [review]

**Comment by alabahuy** [comment]
any update @waffle87 @drashna ?

File: keyboards/jaykeeb/aumz_work/hotswap/info.json
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/hotswap/info.json:1]: please rename this file to `keyboard.json`

File: keyboards/jaykeeb/aumz_work/soldered/info.json
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/soldered/info.json:1]: please rename this file to `keyboard.json`

File: keyboards/jaykeeb/aumz_work/soldered/info.json [hunk 3]
```diff
@@ -0,0 +1,700 @@
...
+    "community_layouts": [
+        "65_ansi_blocker",
+        "65_ansi_blocker_split_bs",
+        "65_ansi_blocker_tsangan",
+        "65_ansi_blocker_tsangan_split_bs",
+        "65_iso_blocker",
+        "65_iso_blocker_split_bs",
+        "65_iso_blocker_tsangan",
+        "65_iso_blocker_tsangan_split_bs"
+    ],
```
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/soldered/info.json:16]: quick tidy
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs", "65_iso_blocker", "65_iso_blocker_split_bs", "65_iso_blocker_tsangan", "65_iso_blocker_tsangan_split_bs"],
```

File: keyboards/jaykeeb/aumz_work/soldered/rules.mk
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/soldered/rules.mk:1]: please delete this file in tandem with `info.json` file rename

File: keyboards/jaykeeb/aumz_work/hotswap/rules.mk
Comment (dunk2k) [inline keyboards/jaykeeb/aumz_work/hotswap/rules.mk:1]: please delete this file in tandem with `info.json` file rename

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Comment by alabahuy** [comment]
oke done @dunk2k

**Comment by alabahuy** [comment]
any update @drashna @drashna @waffle87 ?

**Comment by alabahuy** [comment]
any update @drashna @drashna @waffle87 ?

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**